src/components/state-badge.component.js

```diff
@@ -58,19 +62,9 @@ export const StateBadge = ({
     issueText = translate('issue.main.states.closed', locale);
   }
 
-  let issueStyle = {};
-
-  if (issueState === 'merged') {
-    issueStyle = styles.mergedIssue;
-  } else if (issueState === 'open') {
-    issueStyle = styles.openIssue;
-  } else if (issueState === 'closed') {
-    issueStyle = styles.closedIssue;
-  }
-
   return (
-    <View style={[styles.badge, style, issueStyle]}>
-      <Text style={styles.text}>{issueText}</Text>
-    </View>
+    <Badge issueState={issueState} style={style}>
```

Comment:
I think `style` should be integrated into `Badge`, or we may lose the meaning of styled component.

Reply:
From what I found `style` prop is passed by `repository-section-title` and `issue-description` components.


`Repository-section-title` passed style object:
```
{
  marginLeft: 10,
}
```

`Issue-description` passed style object:
```
{
    alignItems: 'flex-end',
    justifyContent: 'center',
}
```

Reply:
I could not find any examples of passing new style properties to styled component directly. Probably not a good idea? So, what about making `bool` props that would control if the mentioned styles apply?

Reply:
Oh, I think I have found a [solution](https://www.styled-components.com/docs/basics#extending-styles). We could just extend `StateBadge` with the mentioned styles in `issue-description` and `repository-section-title` locally.